NewHorizons/Components/Volumes/StreamingWarpVolume.cs

```diff
@@ -23,6 +24,23 @@ public void Start()
 
         public void FixedUpdate()
         {
+            // Bug report on Astral Codec mod page - Huge lag inside streaming warp volume, possible NRE?
+            if (_probe == null)
+            {
+                _probe = Locator.GetProbe();
+                if (_probe == null)
+                {
+                    NHLogger.LogError($"How is your scout probe null? Destroying {nameof(StreamingWarpVolume)}");
+                    Component.DestroyImmediate(this);
```

Comment:
destroy whole gameobject instead of just the volume?

Reply:
uh yeah good point bc this is a child of the black hole right

Reply:
yuh